buidlerConfigSol4.js

```diff
@@ -1,11 +1,10 @@
 module.exports = {
-  defaultNetwork: "buidlerevm",
```

Comment:
this changes?

Reply:
Not needed, since we only use this file for compilation.

Reply:
Same for buidlerConfigSol5.